### ISSUE TITLE
Work on Morphogenesis and Deep Learning paper

### ISSUE THREAD

**Issue by devoworm** [devoworm/Group-Meetings#81]

**Comment by jesparent** [comment]
@devoworm  any updates on this? is it in open papers?

**Comment by devoworm** [comment]
Yes, this is an open paper: [link](https://drive.google.com/drive/u/2/folders/1lkvKOEYExZfFx7MI47_UWdC7zbKcqM8w)

**Comment by jesparent** [comment]
@devoworm this was talked about again in today's meeting right?

**Comment by Orthogonal-Research-Lab** [comment]
Yes, this was featured in today's meeting. Not fixed home as of yet.
